robo4j-socket-http/src/test/java/com/robo4j/socket/http/test/codec/CameraMessage.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2014, 2023, Marcus Hirt, Miroslav Wengner
+ *
+ * Robo4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Robo4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Robo4J. If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.robo4j.socket.http.test.codec;
+
+import java.util.Objects;
+
+/**
+ * Camera message contains description and image as byte array used for http
+ * codec.
+ *
+ * note: example joystick example
+ *
+ * @author Marcus Hirt (@hirt)
+ * @author Miro Wengner (@miragemiko)
+ */
+public class CameraMessage {
+
+	private String type;
+	private String value;
+	private String image;
+
+	public CameraMessage() {
+	}
```

Comment:
I would add `this(null, null, null)` in order to have always call a single constructor at the end.

Reply:
before I review codecs instantiation part, this was just for JPMS, I hope it explains